models/reviews-models.js

```diff
@@ -1,7 +1,7 @@
 const connection = require("../db/connection");
 
 exports.fetchReviewByReviewId = (id) => {
-  if (typeof Number(id) === "number" && !isNaN(Number(id))) {
+  if (!isNaN(Number(id))) {
```

Comment:
Id want you to instead let SQL handle this. SQL will throw an error if the data type is incorrect so we should let SQL decide if this is an incorrect data type as this is essentially repeating code/functionality